install/kubernetes/cilium/templates/spire/agent/configmap.yaml

```diff
@@ -0,0 +1,54 @@
+{{- if and .Values.auth.mTLS.spire.enabled .Values.auth.mTLS.spire.install.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spire-agent
+  namespace: {{ .Values.auth.mTLS.spire.install.namespace }}
+data:
+  agent.conf: |
+    agent {
+      data_dir = "/run/spire"
+      log_level = "ERROR"
+      server_address = "spire-server"
+      server_port = "8081"
+      socket_path = {{ .Values.auth.mTLS.spire.agentSocketPath | quote }}
+      admin_socket_path = {{ .Values.auth.mTLS.spire.adminSocketPath | quote }}
+      trust_bundle_path = "/run/spire/bundle/bundle.crt"
+      trust_domain = {{ .Values.auth.mTLS.spire.trustDomain | quote }}
+      authorized_delegates = [
+        "spiffe://{{ .Values.auth.mTLS.spire.trustDomain }}/{{ .Values.auth.mTLS.spire.identities.ciliumAgent }}",
+        "spiffe://{{ .Values.auth.mTLS.spire.trustDomain }}/{{ .Values.auth.mTLS.spire.identities.delegatedClient }}",
+      ]
+    }
+
+    plugins {
+      NodeAttestor "k8s_psat" {
+        plugin_data {
+          cluster = {{ .Values.cluster.name | quote }}
+        }
+      }
+
+      KeyManager "memory" {
+        plugin_data {
+        }
+      }
+
+      WorkloadAttestor "k8s" {
+        plugin_data {
+          # Defaults to the secure kubelet port by default.
+          # Minikube does not have a cert in the cluster CA bundle that
+          # can authenticate the kubelet cert, so skip validation.
+          skip_kubelet_verification = true
```

Comment:
this makes it less secure but it does indeed break on some clusters.... maybe we should stand still and think about this one a little better to make it both able to be secure yet easy to install

Reply:
I think it would just be simpler to make it configurable via values.yaml and Minikube users can disable verification, and we can update the guides to set this option for minikube. Cilium CLI can disable it automatically when minikube is detected, like it does for other settings.